### PR TITLE
Update review.php

### PR DESCRIPTION
**Pull request by TheCartpenter**

### CODE REVIEW

File: upload/catalog/controller/product/review.php
Comment (chulcha) [inline upload/catalog/controller/product/review.php:1]: add limit in settings

config_review_limit

Comment (TheCartpenter) [reply upload/catalog/controller/product/review.php:1]: > add limit in settings
> 
> config_review_limit

It is in the admin but is not a mandatory field. Therefore, there are no default value defined either.

Comment (chulcha) [reply upload/catalog/controller/product/review.php:1]: Where?
If it exists, then why is it not used?
And only event on model for change data (((

Comment (TheCartpenter) [reply upload/catalog/controller/product/review.php:1]: Apologize. I thought you were referring to the blog setting limit. Back on topic:

- catalog/controller/account/download.php
- catalog/controller/account/reward.php
- catalog/controller/account/returns.php
- catalog/controller/account/order.php
- catalog/controller/account/subscription.php
- catalog/controller/account/transaction.php
- catalog/controller/cms/comment.php

These files also have user defined value limits, therefore, no ```config_x_limit``` defined either in the catalog.

**Pull request closed without merging**